proto/p4/p4runtime.proto

```diff
@@ -50,88 +78,56 @@ service P4Runtime {
   // Reads 'CounterEntry's.
   rpc CounterRead(CounterReadRequest) returns (stream CounterReadResponse) {
   }
-  // Modifies 'MeterEntry's.
-  rpc MeterWrite(MeterWriteRequest) returns (stream MeterWriteResponse) {
-  }
-  // Reads 'MeterEntry's.
-  rpc MeterRead(MeterReadRequest) returns (stream MeterReadResponse) {
-  }
-  // Represents the bidirectional stream between the controller and the switch
-  // (initiated by the controller), and is managed for the following purposes:
-  // - connection initiation through master arbitration
-  // - indicating switch session liveness: the session is live when switch sends
-  //   a positive master arbitration update to the controller, and is considered
-  //   dead when either the stream breaks or the switch sends a negative update
-  //   for master arbitration
-  // - the controller sending/receiving packets to/from the switch
-  rpc StreamChannel(stream StreamMessageRequest)
-      returns (stream StreamMessageResponse) {
-  }
 }
 
 //------------------------------------------------------------------------------
-message P4ConfigSetRequest {
-  uint64 device_id = 1;
-  config.P4Info p4info = 2;
-  // Target-specific P4 configuration.
-  bytes p4_device_config = 3;
-}
-
-message P4ConfigSetResponse {
-}
-
-message P4ConfigGetRequest {
+message WriteRequest {
   uint64 device_id = 1;
+  Uint128 election_id = 2;
+  repeated Update updates = 3;
 }
 
-message P4ConfigGetResponse {
-  config.P4Info p4info = 1;
-  // Target-specific P4 configuration.
-  bytes p4_device_config = 2;
+message WriteResponse {
 }
```

Comment:
we are not longer returning a repeated field of `google.rpc.Status`?
what's the best practice for returning an error status now?